docs/how_to_make_a_tahoe-lafs_release.org

```diff
@@ -1,80 +1,96 @@
+    - [ ] prefix with SHA256 of tarballs
+    - [ ] release pubkey
+    - [ ] git revision hash
+  - [ ] GPG-signed the release email with release key (write to relnotes.txt.asc)
+
+* publish release artifacts [0/9]
+
+  - [ ] uploaded to PyPI via: twine upload dist/*
+  - [ ] uploaded *.asc to org ~source/downloads/
+  - [ ] test install works properly: pip install tahoe-lafs
+  - [ ] copied the release tarballs and signatures to tahoe-lafs.org: ~source/downloads/
+  - [ ] moved old release out of ~source/downloads (to downloads/old/?)
+  - [ ] updated readthedocs.org (XXX what does this mean? it updates automatically, no?)
+  - [ ] uploaded wheels to https://tahoe-lafs.org/deps/
+  - [ ] updated txAWS-0.2.1.post5.tar.gz (?)
+  - [ ] uploaded release to https://github.com/tahoe-lafs/tahoe-lafs/releases
+
+* check release downloads [0/]
+
+  - [ ] test PyPI via: pip install tahoe-lafs
+  - [ ] https://github.com/tahoe-lafs/tahoe-lafs/releases
+  - [ ] https://tahoe-lafs.org/downloads/
+  - [ ] https://tahoe-lafs.org/deps/
+
+* document release in trac [0/]
+
+  - [ ] closed the Milestone on the trac Roadmap
+
+* unfreeze master branch [0/]
+
+  - [ ] announced on IRC that new PRs will be looked at/merged
+
+* announce new release [0/]
+
+  - [ ] sent release email and relnotes.txt.asc to tahoe-announce@tahoe-lafs.org
+  - [ ] sent release email and relnotes.txt.asc to tahoe-dev@tahoe-lafs.org
+  - [ ] updated Wiki front page: version on download link, News column
+  - [ ] updated Wiki "Doc": parade of release notes (with rev of NEWS.rst)
+  - [ ] make an "announcement of new release" on freshmeat (XXX still a thing?)
   - [ ] make an "announcement of new release" on launchpad
-  - [ ] send out relnotes.txt to:
-    - p2p-hackers@lists.zooko.com
-    - lwn@lwn.net
-    - a Google+ page
-    - cap-talk@mail.eros-os.org
-    - cryptography@metzdown.com
-    - cryptography@randombit.net
-    - twisted-python@twistedmatrix.com
-    - owncloud@kde.org
-    - liberationtech@lists.stanford.edu
-    - the "decentralization" group on groups.yahoo.com
-    - pycrypto mailing list
-    - fuse-devel@lists.sourceforge.net
-    - fuse-sshfs@lists.sourceforge.net
-    - duplicity-talk@nongnu.org
-    - news@phoronix.com
-    - python-list@python.org
-    - cygwin@cygwin.com
-    - The Boulder Linux Users' Group
-    - The Boulder Hackerspace mailing list
-    - cryptopp-users@googlegroups.com
-    - tiddlywiki
-    - hdfs-dev@hadoop.apache.org
-    - bzr
-    - mercurial
-    - http://listcultures.org/pipermail/p2presearch_listcultures.org/
-    - deltacloud
-    - libcloud
-    - swift@lists.launchpad.net
-    - stephen@fosketts.net
-    - Chris Mellor of The Register
-    - nosql@mypopescu.com
-    - The H Open
-    - fans/customers of cleversafe
-    - fans/customers of bitcasa
-    - fans/customers of wuala
-    - fans/customers of spideroak
+  - [ ] tweeted as @tahoelafs
+  - [ ] emailed relnotes.txt to interested third parties (XXX what does this mean?)
```

Comment:
Seems like this whole section of the doc is the list of interested third parties to notify... not sure what this could mean other than announcing to these addresses.

Reply:
As far as I could tell it was just telling them that a new release is available.